src/viur/core/tasks.py

```diff
@@ -466,6 +467,9 @@ def CallDeferred(func: t.Callable) -> t.Callable:
     _target_version: Specify a version on which to run this task.
         By default, a task will be run on the same version where the wrapped method has been called.
 
+    _call_deferred: Calls the @CallDeferred decorated method directly.
+        This is for example necessary, to call a super method which is decorated with @CallDeferred.
+
```

Comment:
Can you please provide a piece of example code in the docstring?

Reply:
I'm not sure what kind of code you image. It's just an argument for the method call

Reply:
I had this case nowhere yet, therefore it should be helpful to others to show such a use-case and how the parameter is used.

Reply:
Something like this?
```py
        class A(Module):
            @CallDeferred
            def task(self):
                ...
        class B(A):
            @CallDeferred
            def task(self):
                super().task(_call_deffered=False)
                ...
            
            @exposed
            def run(self):
                self.task()
```

Reply:
yes, this is helpful to understand how the parameter shall be used.

Reply:
```suggestion
    _call_deferred: Calls the @CallDeferred decorated method directly.
        This is for example necessary, to call a super method which is decorated with @CallDeferred.

    ..  code-block:: python

        # Example for use of the _call_deferred-parameter
        class A(Module):
            @CallDeferred
            def task(self):
                ...

        class B(A):
            @CallDeferred
            def task(self):
                super().task(_call_deferred=False)  # avoid secondary deferred call
                ...
    ..  

```